sounds/fixtures/bulk_uploaders_group.json

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
any reason why you have a single fixture per group instead of putting them all in one file?

Reply:
basically for clarity and because when we load them manually to the database it is more explicit if groups are defined in separate files. however, given that loading a fixture will not duplicate existing objects we could also place them all in one file which is update when new groups are to be added. would you prefer that option @alastair?

Reply:
I think that it makes more sense to include all groups in a single fixture file. So it seems that we'll be OK to add items to it in the future and continue to use loaddata to add a single new group? This sounds fine.

Reply:
Moved all groups to a single fixture 7e920bc042706832a2f9b4df40dd4e53478b4e06